testing/postgresql-xl/APKBUILD

```diff
@@ -0,0 +1,248 @@
+# Maintainer: G.J.R. Timmer <gjr.timmer@gmail.com>
+pkgname=postgresql-xl
+_pkgname=${pkgname/ql/}
+pkgver=9.5r1.4
+pkgrel=0
+pkgdesc="Scalable Open Source PostgreSQL-based Database Cluster"
+url="http://www.postgres-xl.org/"
+arch="all"
+license="BSD"
+# Depends are deliberate, reason: avoid conflict with normal PostgreSQL
+depends="postgresql-xl-client postgresql-xl-libpq"
```

Comment:
@fabled Working on it.
About the naming, I will look in to it, however the product is actually named ```postgresql-xl``` 

Using the depends you override the automatic detection of libpq of the normal libpq package.

I working on gettings details about libpq from the postgres-xl team. to find out if there are differences.